Source/WebKit/WPEPlatform/wpe/wayland/WPEViewWayland.cpp

```diff
@@ -93,17 +113,12 @@ struct DMABufFeedback {
         Data* data { nullptr };
     };
 
-    static std::unique_ptr<DMABufFeedback> create(unsigned size, int fd)
+    static std::unique_ptr<DMABufFeedback> create()
     {
-        auto* data = static_cast<FormatTable::Data*>(mmap(nullptr, size, PROT_READ, MAP_PRIVATE, fd, 0));
-        if (data == MAP_FAILED)
-            return nullptr;
-
-        return makeUnique<DMABufFeedback>(FormatTable(size, data));
+        return makeUnique<DMABufFeedback>();
```

Comment:
Thinking about this again, I guess we could just call makeUnique directly, the create() doesn't add anything in this case.

Reply:
Right, I will remove it before landing.